### DISCUSSION TITLE
Missing bibliography in markdown output?

### DISCUSSION THREAD

**Discussion by krefson** [jgm/pandoc#10012] [Q&A] [answered]
Is it possible to generate a bibliography from LaTeX/bibtex input in the "markdown" variant.  Both my document, and a simple test fail to generate any bibliography in the resulting .md file. 

This works, generating a bibliography with any of "markdown_github", "gfm", "html", "docx" output - just not with "markdown", so I'm confident that I'm following the documentation correctly.

 The documentation suggests that this ought to work, but it does not. Is this a bug?

Just retested with pandoc 3.2.1, and the behaviour is the same.

**Comment by twsh** [comment, marked as answer]
Have you tried 'markdown-citations' as an output format? That's how I do it.

The extension is mentioned in the manual, but I wouldn't have known to use (abuse?) it this way unless somebody told me: https://pandoc.org/MANUAL.html#org-citations

**Reply by krefson** [reply]
Ah, that makes a difference, and generates the bibliography list.

Turning off "citations" to enable a bibliography is sufficiently "unexpected behaviour" that I too failed to guess it would have this effect. 

This has to be either a documentation error or a bug.

**Reply by jgm** [reply]
When pandoc is converting to a markdown format that includes citations, then it will just use its native citation format (and the bibliography isn't generated because, with the native citation system, it will be autogenerated when the markdown file is rendered).  If you say `-t markdown-citations`, then it will render the citations because the native citation format isn't available.  I hope that makes it seem more sensible.

**Reply by iandol** [reply]
I have a pull request to document this (in this case for typst), but we should probably make it more general): https://github.com/jgm/pandoc/pull/9127

**Reply by krefson** [reply]
I would suggest that a note be added to the text documenting the "--citeproc" option.  That currently states _Process the citations in the file, replacing them with rendered citations and adding a bibliography._ flatly and unconditionally, which is what misled me.

**Reply by jgm** [reply]
I have added a note.
Commit b8ed3718b876444dafda7ec369b317fac3517314
